### ISSUE TITLE
drop /allow policy for ip 

### ISSUE THREAD

**Issue by c3ph3us** [ukanth/afwall#513]
can u add options to maintain drop/allow  policy for ip:port ? as simple list 
it's base for any firewall and  afwall is lack of this future :)

**Comment by ukanth** [comment]
yes. I will work on it after adding support for history and xposed. This is on my to-do list for sometime.
